eng/PublishRelease.ps1

```diff
@@ -12,16 +13,31 @@ try
 
    Write-Host "Setting version to $devVersion"
 
-   npm version --no-git-tag-version $devVersion > Out-Null;
+   npm version --no-git-tag-version $devVersion | Out-Null;
    
    $file = npm pack -q;
    $name = [System.IO.Path]::GetFileNameWithoutExtension($file)
 
    Write-Host "Publishing $file"
 
-cmd /c "npx -q publish-release --token $Token --repo autorest.csharp --owner azure --name $name --tag $devVersion --notes=prerelease-build --prerelease --editRelease false --assets $file --target_commitish $Sha 2>&1"
+   #cmd /c "npx -q publish-release --token $Token --repo autorest.csharp --owner azure --name $name --tag $devVersion --notes=prerelease-build --prerelease --editRelease false --assets $file --target_commitish $Sha 2>&1"
 }
 finally
 {
     popd
-}
\ No newline at end of file
+}
+
+
+pushd $RepoRoot
+try
+{
+    # set the version in the root package.json so coverage-push can pick it up
+
+   npm version --no-git-tag-version $devVersion | Out-Null;
```

Comment:
coverage-push picks up the version from the root of the repo but our main package.json is not there, still update the version as a workaround.